config/urls.py

```diff
@@ -5,10 +5,17 @@
 from django.urls import path, re_path
 from django.views import defaults as default_views
 
+from django_otp.admin import OTPAdminSite
 from drf_yasg import openapi
 from drf_yasg.views import get_schema_view
 from rest_framework import permissions
 
+if not settings.DEBUG:
+    admin.site.__class__ = OTPAdminSite
```

Comment:
Should we use the `AdminConfig` for this?

example:

```python
from django.contrib.admin.apps import AdminConfig

class OTPAdminConfig(AdminConfig):
    default_site = 'django_otp.admin.OTPAdminSite'
```

and then install the `OTPAdminConfig`:
```python
INSTALLED_APPS = [
    ...
    'safetransactionservice.admin.OTPAdminConfig',  # replaces 'django.contrib.admin'
    ...
]
```

See https://docs.djangoproject.com/en/dev/ref/contrib/admin/#overriding-the-default-admin-site

Reply:
Didn't know about that way to do that... What way do you think is more explicit?

Reply:
I would say that by using `INSTALLED_APPS` we make it more configurable also in the future in case we want to deploy an instance without the OTP (might be the case also for other teams doing their own deployments).

I would not add right now an environment variable for it but by having it in `INSTALLED_APPS` seems to be a better approach regarding general configuration.

Reply:
Ok, I did it that way 👍🏻